.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Default Issue template
+about: Create an issue to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '…'
+1. Log in as '…'
+1. Click on '…'
+1. Scroll down to '…'
+1. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Software version**
+Tell us which version you're using:
+
+  * from which server URL do you come from? id.test or iddev.test?
+  * checks the version on homepage. Do you see the version as 0.5.0 or a hash as 2493ca6?
```

Comment:
```suggestion
  * server: [id.test.rero.ch][1] or [iddev.test.rero.ch][2]?
  * version: *Check the version on homepage*. Do you see the version number such as `0.5.0` or a hash such as `2493ca6`?

[1]: https://id.test.rero.ch
[2]: https://iddev.test.rero.ch
```
I'm also doing it in rero-ils, in the hope that it will ease a little bit the process.
The reference for the links can be put anywhere you want.